src/shaders/interpolation/shaders.interpolation.trilinear.js

```diff
@@ -103,6 +103,10 @@ void ${this._name}(in vec3 currentVoxel, out vec4 dataValue, out vec3 gradient){
   trilinearInterpolation(normalizedPosition, interpolatedValue ,v000, v100, v001, v101, v010,v110, v011,v111);
   dataValue = interpolatedValue;
 
+  if (gradient.x == 1.) { // skip gradient calculation for slice helper
```

Comment:
That is OK-ish for now but now but ideally we would have a different shader file that only computes the gradients.